### PR TITLE
Yueyue/notification

### PR DESCRIPTION
**Pull request by CleoZhang**
#118 and #117 are fixed

### CODE REVIEW

File: app/src/main/java/org/cpen321/discovr/CoursesFragment.java [hunk 1]
```diff
@@ -59,26 +59,107 @@ public View onCreateView(LayoutInflater inflater,  ViewGroup container,  Bundle
...
+//        }else if( 10 <= hrs &&  hrs <= 22){
+//            result = Integer.parseInt(String.valueOf(startTime).substring(0,4));
+//            return result;
+//        }
+//        return -1;
+//    }
+
+//    private String formatDow(int dow) {
+//        if(dow == 1 || dow == 7){
+//            return "Weekend";
```
Comment (ValRat) [inline app/src/main/java/org/cpen321/discovr/CoursesFragment.java]: What's this for?

Comment (CleoZhang) [inline app/src/main/java/org/cpen321/discovr/CoursesFragment.java]: something I tried to format time. not helpful anymore.

Comment (ValRat) [inline app/src/main/java/org/cpen321/discovr/CoursesFragment.java]: Can we remove it?

Comment (CleoZhang) [inline app/src/main/java/org/cpen321/discovr/CoursesFragment.java]: sure, i ll remove it then

**Pull request merged**